CMakeLists.txt

```diff
@@ -221,6 +218,7 @@ message(STATUS "CPU architecture: ${CMAKE_HOST_SYSTEM_PROCESSOR}")
 
 if(FF_GPU_BACKEND STREQUAL "hip_cuda" OR FF_GPU_BACKEND STREQUAL "hip_rocm")
   set(ROCM_PATH "/opt/rocm" CACHE STRING "Default ROCM installation directory.")
+  set(GPU_TARGETS "gfx1036;gfx1100" CACHE STRING "The GPU TARGETs")
```

Comment:
where is this variable being used?

Reply:
if this variable is still needed, we should probably set it equal to the FF_HIP_ARCH variable from `cmake/hip.cmake`

Reply:
it's a rocm flag to compile .cpp file in flexflow, I think it's required, and the FF_HIP_ARCH is used to set legion. I tried removing the GPU_TARGETS, the hipLaunchKernelGGL gives a segmentation fault.

Reply:
I moved it to hip.cmake now